src/server/network/network.ts

```diff
@@ -1,5 +1,6 @@
 import dgram = require('dgram');
```

Comment:
These could be `import ... from`:

```suggestion
import dgram from 'dgram';
```